# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/animation/KeyframeUtils.h
# Commit: 63603267402e
# Full Hash: 63603267402ee93d6551ef60c35191d99aaee06b
# Author: Zach Hoffman <zach@zrhoffman.net>
# Date: 2023-12-18 17:17:57
# Regressor Bug: 1846516
# File Overlap Count: 3
# Description:
#   Bug 1846516 - [css-properties-values-api] Introduce AnimatedPropertyID/AnimatedPropertIDSet. r=firefox-animation-reviewers,firefox-style-system-reviewers,layout-reviewers,emilio,hiro
#   
#   This will make possible to animate custom properties on Gecko side. For now, the
#   animation code keeps only dealing with nsCSSPropertyID, so behavior is unchanged.
#   
# ==============================================================================

diff -r 753bc39f59ff -r 63603267402e dom/animation/KeyframeUtils.h
--- a/dom/animation/KeyframeUtils.h	Mon Dec 18 09:24:07 2023 +0000
+++ b/dom/animation/KeyframeUtils.h	Mon Dec 18 09:24:08 2023 +0000
@@ -16,6 +16,7 @@
 class JSObject;
 
 namespace mozilla {
+struct AnimatedPropertyID;
 struct AnimationProperty;
 class ComputedStyle;
 
@@ -102,7 +103,7 @@
    *                  if the property is animatable or not.
    * @return true if |aProperty| is animatable.
    */
-  static bool IsAnimatableProperty(nsCSSPropertyID aProperty);
+  static bool IsAnimatableProperty(const AnimatedPropertyID& aProperty);
 };
 
 }  // namespace mozilla